README.md

```diff
@@ -82,10 +82,18 @@ Short description.
 
   Collaborated with NASA to develop a shower system that uses 90% less water. The shower has a purification system that makes it possible to reuse the same batch of water. The company now wants to make this technology - initially developed for space - available to consumers.
 
+- [Seabin Project](https://seabinproject.com/) | Australia | -
+
+  To provide practical and tangible solutions to reduce the plastics in our oceans which is one of the world’s greatest problems.
```

Comment:
I think the description could be improved to tell better what they're doing, and how far they're into the development of their solution. What about this instead:

```
Their mission is to reduce plastic in the oceans through a floating rubbish bin that 
collect pollutants on the water surface. As of October 2019 they have deployed 
719 Seabins and are capturing almost 3 tonnes of rubbish per day.
```

Reply:
Agreed. Will make the changes